caffeine@patapon.info/schemas/org.gnome.shell.extensions.caffeine.gschema.xml

```diff
@@ -31,15 +37,10 @@
         <summary>Enable when a fullscreen application is running</summary>
         <description>Enable when a fullscreen application is running</description>
     </key>
-    <key type="b" name="control-nightlight">
-        <default>false</default>
+    <key name="control-nightlight" enum="org.gnome.shell.extensions.caffeine.control-nightlight">
+        <default>"never"</default>
```

Comment:
Wouldn't this be a problem while upgrading ? Ie old setting type won't match new setting type

Reply:
OK, so what happens if the value is already set with the wrong type is that it will be deleted and recreated using the correct type. This means the user's setting is lost.
I can look to rewrite this particular part to convert the user's current setting.
I will most likely need to rename the new setting to something else because reassigning the type will lead to the aforementioned behaviour upon loading the modified schema.

Reply:
I've addressed this in the latest commit.

Reply:
Thanks a lot!